src/librustc/mir/interpret/error.rs

```diff
@@ -400,139 +475,131 @@ impl fmt::Display for InterpError<'_> {
 impl fmt::Debug for InterpError<'_> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         use InterpError::*;
+        use UnsupportedInfo::*;
         match *self {
-            PointerOutOfBounds { ptr, msg, allocation_size } => {
+            Unsupported(PointerOutOfBounds { ptr, msg, allocation_size }) => {
```

Comment:
Please have one `Debug` implementation for each of the `Info` types -- then you do not have to match on two levels of enums at once.

Reply:
Yeah, I added a `Debug` implementation for all `Info` types except `UnsupportedInfo`. Will add that.

Reply:
Done.